app/javascript/orangelight/availability.es6

```diff
@@ -133,10 +133,10 @@ export default class AvailabilityUpdater {
   // search results
   process_result(record_id, holding_records) {
     for (let holding_id in holding_records) {
-      if (holding_id.match(/[a-zA-Z]\$[a-zA-Z]/) || holding_id.startsWith('fake_id_')) {
+      if (holding_id.match(/[a-zA-Z]\$[a-zA-Z]/)) {
         // In this case we cannot correlate the holding data from the availability API
         // (holding_records) with the holding data already on the page (from Solr).
-        // In this case we set all of them to "Check record" because we can get this
+        // In this case we set all of them to "View record for Full Availability" because we can get this
         // information in the Show page.
         const badges = $(`*[data-availability-record='true'][data-record-id='${record_id}'] span.availability-icon`);
         badges.text("View record for Full Availability")
```

Comment:
we would need to review the 'View record for full availability' after we swap the index with the temp locations.

Reply:
https://github.com/pulibrary/orangelight/issues/2885